services/app/src/contexts/ws-context.jsx

```diff
@@ -23,6 +23,9 @@ const WebSocketsContextProvider = ({ children }) => {
 
     webSocket.addEventListener('message', (event) => {
       console.log('message: ', event.data);
+      if (JSON.parse(event.data).type === 'PlayerDisconnected'){
```

Comment:
Does this reload the application for all the players, when one of the players disconnects?

Reply:
Yes. I have just tested it.

Reply:
I dont think this is the proper way of handling player list update.

Application reload is potentially a costly operation, we figure out a way of removing player from the list without it.

If players list is stored somewhere in the frontend, we could filter that list to exclude the player that was disconnected. If the WS event about disconnected player does not include information about player that was disconnected, we should include it in the event payload.